codegen/templates/templates.go

```diff
@@ -627,8 +627,41 @@ func WriteRuntimeObjectWrapper(metadata RuntimeObjectWrapperMetadata, out io.Wri
 	return templateRuntimeObject.Execute(out, metadata)
 }
 
+// goKeywords is the set of Go reserved keywords that cannot be used as package names
+var goKeywords = map[string]struct{}{
```

Comment:
why not just a slice + `slices.Contains`? more concise & compact and I can't imagine it's a measurable perf impact :)